pillar/devstack.sls

```diff
@@ -1,151 +1,77 @@
 #!jinja|yaml
 
-{# TODO: Incorporate this for deploying xqueue to a separate instance
-{# This needs to be set to a domain that is addressable by the xqueue server #}
-{# because it is used when constructing the callback URL #}
-{# EDXAPP_LMS_SITE_NAME: lms.service.consul #}
-{# EDXAPP_CMS_SITE_NAME: cms.service.consul #}
-{# EDXAPP_XQUEUE_URL: http://xqueue.service.consul #}
+{% set business_unit = 'mitodl' %}
+{% set purpose = 'devstack' %}
+{% set environment = 'dev' %}
+{% set purpose_data =  %}
+{% set purpose_suffix = 'devstack' %}
 
-{# Move all following pillar data under a top-level key of `ansible_vars` #}
-{# Use subkeys for the respective apps/playbooks (e.g. `forum`, `xqueue`, etc.) #}
+{% set xqueue_rabbitmq_username = salt.random.get_str(12) %}
```

Comment:
For usernames this is valid, but it may be a bit more user-friendly to have descriptive strings for the usernames, in case they get served up in any sort of error message. I'm not sure if they ever would, but there's no harm in having it.